integration-cli/docker_cli_import_test.go

```diff
@@ -39,3 +42,60 @@ func (s *DockerSuite) TestImportDisplay(c *check.C) {
 	}
 
 }
+
+func (s *DockerSuite) TestImportFile(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "run", "--name", "test-import", "busybox", "true")
+	out, _, err := runCommandWithOutput(runCmd)
+	if err != nil {
+		c.Fatal("failed to create a container", out, err)
+
+	}
+
+	temporaryFile, err := ioutil.TempFile("", "exportImportTest")
+	if err != nil {
+		c.Fatal("failed to create temporary file", "", err)
+	}
+
+	runCmd = exec.Command(dockerBinary, "export", "test-import")
+	runCmd.Stdout = bufio.NewWriter(temporaryFile)
+
+	_, err = runCommand(runCmd)
+	defer os.Remove(temporaryFile.Name())
+	if err != nil {
+		c.Fatal("failed to export a container", out, err)
+	}
+
+	runCmd = exec.Command(dockerBinary, "import", temporaryFile.Name())
+	out, _, err = runCommandWithOutput(runCmd)
+	if err != nil {
+		c.Fatal("failed to import a container", out, err)
+	}
+
+	if n := strings.Count(out, "\n"); n != 1 {
+		c.Fatalf("display is messed up: %d '\\n' instead of 1:\n%s", n, out)
+	}
+	image := strings.TrimSpace(out)
+	defer deleteImages(image)
```

Comment:
you can delete this defer as the suite takes care of deleting images here https://github.com/docker/docker/blob/master/integration-cli/check_test.go#L37